src/test/CMakeLists.txt

```diff
@@ -157,11 +157,12 @@ if(ENABLE_WALLET)
   add_subdirectory(${PROJECT_SOURCE_DIR}/src/wallet/test wallet)
 endif()
 
-if(WITH_MULTIPROCESS)
+if(ENABLE_IPC)
   target_link_libraries(bitcoin_ipc_test
     PRIVATE
       core_interface
       univalue
+      Boost::headers
```

Comment:
In abdf3cb6456f3f79b3130d2001bf780deadccf87: 
> fails due to warnings in boost headers treated as errors, reported in

I might be missing something, but can't see anything Boost related in either of the reports?